### ISSUE TITLE
Cannot download FFMPEG exe when frozen on Windows

### ISSUE THREAD

**Issue by paulmueller** [imageio/imageio#271]
I am not sure if this issue is imageio-related. It seems like imageio is not able to download the ffmpeg binary if it is running in a frozen application on a Windows Virtual Machine hosted by a Mac. Note that this issue does not occur on a regular Windows installation. I do not have the resources to thoroughly test this issue or reproduce it.

To reproduce the error:
1. Download the installer for ShapeOut 0.7.7: https://github.com/ZELLMECHANIK-DRESDEN/ShapeOut/releases/tag/0.7.7
2. Make sure ffmpeg is not installed on the system and imageio did not cache any ffmpeg binaries.
3. Execute ShapeOut. If it does not find ffmpeg, a dialog box will be displayed asking the user to download it. (`imageio.plugins.ffmpeg.download()` will be called).

Then the following error occurs: 

https://github.com/ZELLMECHANIK-DRESDEN/ShapeOut/issues/165
```
Traceback (most recent call last):
File "ShapeOut.py", line 66, in
File "ShapeOut.py", line 38, in main
File "gui\frontend.py", line 175, in InitRun
File "site-packages\imageio\plugins\ffmpeg.py", line 58, in download
File "site-packages\imageio\core\fetching.py", line 121, in get_remote_file
File "site-packages\imageio\core\fetching.py", line 177, in _fetch_file
IOError: Unable to download 'ffmpeg-win32-v3.2.4.exe'. Perhaps there is a no internet connection? If there is, please report this problem.
```

**Comment by almarklein** [comment]
My first guess is that it might be due to certain libs not being frozen along, such as those to do https requests.

**Comment by almarklein** [comment]
I was going to suggest that frozen apps should probably ship ffmpeg if they need it, and then saw that this is exactly what you did. I'll leave this open though, in case someone wants to have a go at fixing it some day.

**Comment by almarklein** [comment]
This should be fixed with #425, in the sense that there will be no more download mechanism for ffmpeg. Though we need a solution for frozen apps. I created https://github.com/imageio/imageio-ffmpeg/issues/5 to track this.

**Comment by paulmueller** [comment]
I don't know if this has actually ever been a real issue. On a regular windows machine, this always worked. Maybe some SSL certificates were missing or PyInstaller fixed it somehow.
